source/_integrations/generic_thermostat.markdown

```diff
@@ -89,6 +89,11 @@ precision:
   required: false
   type: float
   default: "`0.1` for Celsius and `1.0` for Fahrenheit."
+target_temp_step:
+  description: "The desired step size for setting the target temperature. Supported values are `0.1`, `0.5` and `1.0`."
+  required: false
+  type: float
+  default: "equal to `precision`"
```

Comment:
Not sure what the guidelines are here, but `precision` `default` ends with a `.`